libs/graph-lib/src/services/encryption.service.ts

```diff
@@ -17,53 +16,49 @@ export class EncryptionResult {
 }
 
 @Injectable()
-export class EncryptionService {
+export class EncryptionService implements OnModuleInit {
   private readonly logger: Logger;
 
-  private readonly suite: CipherSuite;
+  private suite: CipherSuite;
 
   private encryptionKey: CryptoKeyPair;
 
   constructor(@Inject(graphCommonConfig.KEY) private readonly config: IGraphCommonConfig) {
     this.logger = new Logger(this.constructor.name);
+  }
+
+  async onModuleInit() {
```

Comment:
`this.suite` can still be `private readonly` and initialized in constructor, since it doesn't need any async initialization.